### ISSUE TITLE
Assertion failure in SDL on M1 Pro

### ISSUE THREAD

**Issue by def-** [ddnet/ddnet#4434]
```
Assertion failure at SDL_GetWindowFlags_REAL (/home/deen/isos/ddnet/debian6/root/mac64/SDL2-2.0.16/src/video/SDL_video.c:1905), triggered 1 time:
'window && window->magic == &_this->window_magic'
```

**Comment by def-** [comment]
Starts anyway after clicking on Ignore All, seems generally usable.
There's the problem of having to switch to another window and back to get mouse to stick

**Comment by def-** [comment]
>  There's the problem of having to switch to another window and back to get mouse to stick

This seems to be fixed with SDL 2.0.18
